### PR TITLE
Relax mp3 detection algorithm

### PR DESCRIPTION
**Pull request by buzinas**
Refs https://github.com/closeio/closeio/issues/8302.

### CODE REVIEW

File: filevalidator.js [hunk 1]
```diff
@@ -11,7 +11,13 @@ function(_) {
             // MPEG-1 Layer 3 file without an ID3 tag or with an ID3v1 tag (which's appended at the end of the file)
             [0xFF, 0xFB],
             // MP3 file with an ID3v2 container
-            [0x49, 0x44, 0x33]
+            [0x49, 0x44, 0x33],
+            // Other MP3 files (FF Fx and FF Ex – they may cause false-positives)
+            // Headers got from https://www.garykessler.net/library/file_sigs.html
```
Comment (wojcikstefan) [inline filevalidator.js]: taken from

File: package.json [hunk 2]
```diff
@@ -1,11 +1,8 @@
 {
   "name": "filevalidator",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "File signature validation in JavaScript",
   "main": "filevalidator.js",
-  "scripts": {
```
Comment (wojcikstefan) [inline package.json:6]: Was this unnecessary? I don't see any tests added to the PR.

**Review by wojcikstefan — APPROVED** [review]
LGTM, though I haven't tested it.

File: package.json (hunk 2 above)
Comment (philfreo) [reply package.json:6]: @wojcikstefan I think his point is "why was it necessary"

Comment (buzinas) [reply package.json:6]: This is added by `npm init` and it's unnecessary. It was just a clean up.

File: filevalidator.js [hunk 3]
```diff
@@ -11,7 +11,13 @@ function(_) {
...
+            [0xFF, 0xE0], [0xFF, 0xE1], [0xFF, 0xE2], [0xFF, 0xE3], [0xFF, 0xE4],
+            [0xFF, 0xE5], [0xFF, 0xE6], [0xFF, 0xE7], [0xFF, 0xE8], [0xFF, 0xE9],
```
Comment (philfreo) [inline filevalidator.js]: Isn't `x` in the external site referring to a hex number? So e.g. 0xEA - 0xEF are missing still ? Or am I thinking about this wrong?

Comment (buzinas) [reply filevalidator.js]: Hm, you're right, good point! Let me fix!

Comment (buzinas) [reply filevalidator.js]: Done!

**Review by philfreo — APPROVED** [review]

**Pull request merged**